vignettes/data_overview.Rmd

```diff
@@ -128,8 +128,6 @@ ggplot() +
   scale_y_continuous(trans='log10') +
   ylab('log(median prey density) (count/L)') +
   xlab('month') +
```

Comment:
can we do theme minimal on all the plots
Also more straightforward for the plots if you put the aes in the ggplot and than add the box plot to it 
easier if we want to make updates in the future

Reply:
please apply to all plots